kinto-remote-settings/src/kinto_remote_settings/changes/views.py

```diff
@@ -40,8 +40,7 @@ def __init__(self, request):
     def timestamp(self):
         if not self._entries():
             return core_utils.msec_time()
-        max_value = max([e["last_modified"] for e in self._entries()])
-        return max_value
+        return max([e["last_modified"] for e in self._entries()])
```

Comment:
The thing I like about this old version is that it makes it easier to throw a breakpoint in there before returning

```diff
  max_value = max([e["last_modified"] for e in self._entries()])
+ breakpoint()
  return max_value
```

is there a way to achieve something similar with this new linting rule?

Reply:
I hadn't thought of that. I could definitely revert 8264b5eb7450bf477dea46e15c8d57cb16273683